### PR TITLE
Zh trans2018 05 30 say hello to discuss kubernetes.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+作者: Jorge Castro (Heptio)
+
+<!-- 
+
+Communication is key when it comes to engaging a community of over 35,000 people in a global and remote environment. Keeping track of everything in the Kubernetes community can be an overwhelming task. On one hand we have our official resources, like Stack Overflow, GitHub, and the mailing lists, and on the other we have more ephemeral resources like Slack, where you can hop in, chat with someone, and then go on your merry way. 
+
+-->
+
+成为在全球范围有超过 35,000 人的社区时沟通是非常关键的。 跟踪 Kubernetes 社区中的所有内容可能是一项艰巨的任务。 一方面，我们有官方资源，如 Stack Overflow，GitHub 和邮件列表，另一方面，我们有更多实时的资源，如 Slack，你可以在那里跳，与某人聊天，然后继续按你喜欢的方式去做。
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: ephemeral 瞬时性的

hop in 加入进去

go on your merry way 各走各路

File: content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+While our current assortment of communication channels are valuable in their own rights, we found that there was still a gap between email and real time chat. Across the rest of the web, many other open source projects like Docker, Mozilla, Swift, Ghost, and Chef have had success building communities on top of Discourse, an open source discussion platform. So what if we could use this tool to bring our discussions together under a modern roof, with an open API, and perhaps not let so much of our information fade into the ether? There's only one way to find out: Welcome to discuss.kubernetes.io
+
+-->
+
+Slack 非常适合随意和及时的对话，并与其他社区成员保持联系，但未来很难轻易引用通信。此外，在35,000名参与者中提问并得到回答很难。邮件列表在有问题尝试联系特定人群并且想要跟踪大家的回应时非常有用，但是对于大量人员来说可能是麻烦的。 Stack Overflow 和 GitHub 非常适合在涉及代码的项目或问题上进行协作，并且如果在将来要进行搜索也很有用，但某些主题如“你最喜欢的 CI/CD 工具是什么”或“[Kubectl提示和技巧](http://discuss.kubernetes.io/t/kubectl-tips-and-tricks/192)“在那里是没有意义的。
```
Comment (tengqm) [inline content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: 这种大段翻译时，请考虑按句子换行。只要句子中间没有多余的空行，不会影响最终的渲染结果。但是增加换行之后，会很方便 review。

File: content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+虽然我们目前的各种沟通渠道对他们自己来说都很有价值，但我们发现电子邮件和实时聊天之间仍然存在差距。在网络的其他部分，许多其他开源项目，如 Docker，Mozilla，Swift，Ghost 和 Chef，已经成功地在[Discourse](http://www.discourse.org/features)之上构建社区，一个开放的讨论平台。那么，如果我们可以使用这个工具将我们的讨论结合在一个平台下，使用开放的API，或许也不会让我们的大部分信息消失在网络中呢？只有一种方法可以找到：欢迎来到[discuss.kubernetes.io](http://discuss.kubernetes.io)
```
Comment (tengqm) [inline content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: 多个词语、子句之间排比时，请使用中文的顿号。

**Comment by congfairy** [comment]
@tengqm 我改完了 请review 谢谢

File: content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md (hunk 2 above)
Comment (markthink) [reply content/zh/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md]: 同意

**Pull request merged**